R/tbl_propdiff.R

```diff
@@ -0,0 +1,365 @@
+#' Calculating unadjusted and adjusted differences in rates
+#'
+#' This function calculates the unadjusted or adjusted difference in rates with confidence interval.
+#'
+#' @param data A data frame
+#' @param outcome A vector of outcome variable names. Outcome variables can be numeric, character or factor, but must have two and only two non-missing levels
+#' @param predictor The name of the predictor (stratifying) variable. This can be numeric, character or factor, but must have two and only two non-missing levels
+#' @param covariates A vector of variable names that the difference in rates will be adjusted for. The p-value displayed will be from a multivariable logistic regression model containing these covariates. The default option is `NULL` which calculates an unadjusted difference in rates.
```

Comment:
Can we use formula notation to keep the syntax consistent with `tbl_ancova()` and `tbl_uvregerssion()`?

Reply:
updated to use formula notation